Meet cmake error when building nvtx3 on Jetson Orin

Hi!

I meet a cmake error when building nvtx3 on Jetson Orin, which is

```
CMake Error at imports/NVTX/nvtxImportedTargets.cmake:54 (set_target_properties):
  INTERFACE_LIBRARY targets may only have whitelisted properties.  The
  property "VERSION" is not allowed.
Call Stack (most recent call first):
  imports/NVTX/CMakeLists.txt:33 (include)


CMake Error at imports/NVTX/nvtxImportedTargets.cmake:64 (set_target_properties):
  INTERFACE_LIBRARY targets may only have whitelisted properties.  The
  property "VERSION" is not allowed.
Call Stack (most recent call first):
  imports/NVTX/CMakeLists.txt:33 (include)
```

The version of cmake is 3.16.3
By the way, I have sucessfully built nvtx3 on my laptop and the verison of cmake is 3.16.3

--- a/c/CMakeLists.txt
+++ b/c/CMakeLists.txt
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #=============================================================================
-cmake_minimum_required(VERSION 3.10)
+cmake_minimum_required(VERSION 3.19)
 
 # This file is typically used by calling add_subdirectory on this directory or add
 # this directory as an external package.  In this case, the NVTX library targets

--- a/c/nvtxImportedTargets.cmake
+++ b/c/nvtxImportedTargets.cmake
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #=============================================================================
-cmake_minimum_required(VERSION 3.10)
+cmake_minimum_required(VERSION 3.19)
 
 set(NVTX3_VERSION 3.1.0)
 

